src/main/java/com/playmonumenta/networkrelay/RemotePlayerGeneric.java

```diff
@@ -11,7 +12,7 @@
 public class RemotePlayerGeneric extends RemotePlayerAbstraction {
 	protected final String mServerType;
 
-	protected RemotePlayerGeneric(String serverType, String serverId, UUID uuid, String name, boolean isOnline, boolean isHidden) {
+	protected RemotePlayerGeneric(String serverType, String serverId, UUID uuid, String name, boolean isOnline, @Nullable Boolean isHidden) {
```

Comment:
Shouldn't this be boolean instead of Boolean. If not feel free to resolve this comment.

Reply:
Intended

Reply:
a primitive type cannot be nullable if I remember right, so I can understand the use of a boolean.

Reply:
Can confirm - this is for use in stuff that isn't Minecraft or a Minecraft proxy, like a Discord bot that might have no clue if a player is vanished or not. Using null here means "I dunno, just use whatever the other server types said".